classes/cache/CacheMemcached.php

```diff
@@ -79,10 +79,7 @@ public function connect()
             $this->memcached->addServer($server['ip'], $server['port'], (int) $server['weight']);
         }
 
-        $versions = $this->memcached->getVersion();
-        $this->is_connected = is_array($versions)
```

Comment:
Are you sure that `$versions` is always an array? Sure sure sure 😄  ?

Reply:
:tada: https://www.php.net/manual/fr/memcached.getversion.php :tada: